packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -433,6 +434,7 @@ class _ExpansionPanelListState extends State<ExpansionPanelList> {
   @override
   Widget build(BuildContext context) {
     final List<MergeableMaterialItem> items = <MergeableMaterialItem>[];
+    final MaterialLocalizations localizations = MaterialLocalizations.of(context);
```

Comment:
Maybe move this into the if since it is only used in that case?

Reply:
fixed